src/Relations/BelongsToThrough.php

```diff
@@ -394,6 +357,8 @@ public function getQualifiedFirstLocalKeyName()
     /**
      * Make a new related instance for the given model.
      *
+     * @inheritDoc
+     *
      * @param \Illuminate\Database\Eloquent\Model $parent
      * @return \Illuminate\Database\Eloquent\Model
      */
```

Comment:
this should be replaced

Reply:
Only `/** @inheritDoc */` result in:

```
  364    Method Znck\Eloquent\Relations\BelongsToThrough::newRelatedInstanceFor() has no return type        
         specified.                                                                                         
         🪪  missingType.return      
```

Reply:
Ah, it looks like it's not actually defined on the root `Relation`:

![image](https://github.com/user-attachments/assets/8a801178-81f8-4fd6-9d94-7a388cd9b454)

In that case just copy the above docs